packages/app/app/stacks/stacksOrders.tsx

```diff
@@ -231,6 +231,7 @@ export const StackOrders = ({ chainId, address }: StackOrdersProps) => {
                           stacks.orders,
                           stacks.sort
                         )}
+                        refetchAllOrders={fetchAllOrders}
```

Comment:
why no use `fetchAllOrders` instead of  `refetchAllOrders`?
cause it's seems to fetchAllOrders and not necessarily has the logic for re-fetching.

Reply:
Looks like a refetch, right? Also we are using refetchStacks with the same logic

Reply:
Exactly as Leo suggested, the idea for this prop is that, when we successfully cancel a stack and close the modal, we re-fetch the needed data:
* The total orders count used for the tabs count (and the "All stacks count")
* The displayable orders per page

Reply:
but we're passing it a `fetchAllOrders` function, I would keep the same name. As the function fetches the orders. Where we use it is the re-fetch thing, but would keep the same name.

Reply:
Ah I get the semantic you suggest now, I'm ok with your suggestion, will push it in a bit.